src/pages/components/lead-space-search.mdx

```diff
@@ -13,129 +13,161 @@ import ComponentFeedback from 'components/ComponentFeedback';
 
-## Options
+## Anatomy
+
+<Row>
 
-### With adjacent theme search
+<Column colMd={8} colLg={8}>
 
-Lead space search provides the option to wrap the search field and search field container into an adjacent theme. You can use the adjacent themes pairs White/Gray10, Gray10/White, Gray90/Gray100 and Gray100/Gray90.
+![A visual example of the anatomy of the lead space search component](../../images/component/lead-space-search/lead-space-search-anatomy.png)
 
-![A visual example of adjacent themes used in the lead space search compoonent](../../images/component/lead-space-search/lead-space-search-variant-adjacent-themes.jpg)
+<Caption>Anatomy of the lead space search component</Caption>
+
+</Column>
+
+</Row>
+
+1. **Heading:** Customizable heading for the lead space search component.
+2. **Subheading:** Optional supporting copy under the heading.
+3. **Search:** Search field component displayed prominently.
+
+## Modifiers
+
+### Adjacent theme
+
+Lead space search provides the option to display the heading and subheading container as one theme, and the search field as an adjacent theme. When enabled, the search field container will automatically adopt the adjacent theme which pairs with the theme in the heading and subheading container.
+
+This is useful when you wish to visually separate the two containers while more closely associating the search with the results visualized on the page.
+
+<Row>
+
+<Column colMd={6} colLg={8}>
+
+![A visual example of adjacent themes used in the lead space search component](../../images/component/lead-space-search/lead-space-search-modifier-adjacent-themes.png)
 
 <Caption>
   Examples of lead space search using adjacent themes. From top to bottom:
   White/Gray 10, Gray 10/White, Gray 90/Gray 100, and Gray 100/Gray 90.
 </Caption>
 
-## Anatomy
+</Column>
 
-![A visual example of the anatomy of the lead space search component](../../images/component/lead-space-search/lead-space-search-anatomy.jpg)
+</Row>
 
-<Caption>Anatomy of the lead space search component</Caption>
+### Heading highlight
 
-1. **Heading (required):** Customizable heading for the lead space search component.
-2. **Subheading (optional):** Emphasize the heading by adding a subheading under it.
-3. **Search (required):** Search field component displayed prominently.
-4. **Typeahead (required):** Carbon typeahead component that displays increasingly accurate suggestions for users as they type in the search field.
-5. **Background media (optional)**
+Like other lead spaces, lead space search's heading can be modified to highlight a word or phrase. This can be used to highlight product names, industries, or particular value. The highlighted word or phrase can be anywhere in the heading, but it should only ever appear once in a single lead space. This only applies to the heading, and does not extend to any other lead space search element.
```

Comment:
```suggestion
Like other lead spaces, lead space search's heading can be modified to highlight a word or phrase. This can be used to highlight product names, industries, or a particular value. The highlighted word or phrase can be anywhere in the heading, but it should only ever appear once in a single lead space. This only applies to the heading, and does not extend to any other lead space search element.
```